### ISSUE TITLE
Wrong input on button click in Calculator

### ISSUE THREAD

**Issue by sairish2001** [sairish2001/makesmatheasy.github.io#20]
When user clicks **log10(** it displays **sqrt(** but should display **log10(** in input field
![Untitled](https://user-images.githubusercontent.com/60106112/110218604-a458f700-7ee0-11eb-9baa-f0ab9c6523d0.png)

Also on clicking More
![Untitledd](https://user-images.githubusercontent.com/60106112/110218634-d10d0e80-7ee0-11eb-8579-898da5b5f492.png)
Be make sure that on clicking √ it must display **sqrt(** not **√** on input field

**Comment by DEEPIKA-SUK** [comment]
I am a participant of GSSOC 2021. I want to contribute to this issue and work on the same. Please could you assign me this issue as a part of GSSOC 2021.
Thank you.

**Comment by sairish2001** [comment]
Feel free to ask your doubts, if any

**Comment by DEEPIKA-SUK** [comment]
Thank you sir!

**Comment by sairish2001** [comment]
[Read](https://github.com/sairish2001/makesmatheasy.github.io/blob/main/CONTRIBUTING.md)

**Comment by DEEPIKA-SUK** [comment]
![Screenshot](https://user-images.githubusercontent.com/64724039/110344654-b909cc00-8053-11eb-8822-b4dac86580a2.png)
Please review my work and please could you tell me when should I create the pull request.

**Comment by sairish2001** [comment]
Is your code is working fine with **√** symbol in input field?

**Comment by DEEPIKA-SUK** [comment]
Yes sir. I have provided this example as the input in the above attached screenshot and it's showing the correct output.

**Comment by sairish2001** [comment]
log10( is working fine?

**Comment by DEEPIKA-SUK** [comment]
![Screenshot (112)](https://user-images.githubusercontent.com/64724039/110347087-42ba9900-8056-11eb-8e17-d1dba5bb474f.png)
![Screenshot (113)](https://user-images.githubusercontent.com/64724039/110347090-43ebc600-8056-11eb-8263-e50894b4e9b3.png)

**Comment by DEEPIKA-SUK** [comment]
Sir kindly review these examples.

**Comment by sairish2001** [comment]
Great Work!
Make a PR

**Comment by sairish2001** [comment]
Please do mention this issue while making PR, as it's a good practice

**Comment by DEEPIKA-SUK** [comment]
As a title sir?

**Comment by sairish2001** [comment]
Have you changed something in index.html file?

**Comment by DEEPIKA-SUK** [comment]
Yes sir

**Comment by DEEPIKA-SUK** [comment]
I am facing issues while creating pull request due to some conflicts in index.html file.
I made changes in only one line in index.html file.

**Comment by DEEPIKA-SUK** [comment]
And some changes in js file.

**Comment by sairish2001** [comment]
can you please tell the changes

**Comment by DEEPIKA-SUK** [comment]
Yes sir,
onclick="document.getElementById('txt').value+=this.innerText+(this.innerText=='√'?'(':'');calsol(document.getElementById('txt').value);"
I made this change in index file for sqrt one.

**Comment by sairish2001** [comment]
Actually while implementing the Radian to Degree conversion code, one of our contributor made changes into index.html and simplecal.js files. i.e. instead of `inline onclick=" "` he adds `eventlistner()` in simplecal.js.

That's why PR is creating conflicts.

I would suggest you to made these changes to newer code. Sorry for the inconvenience. It's not difficult just instead of inline onclick implement in `simplecal.js` with eventlistner

**Comment by DEEPIKA-SUK** [comment]
Sorry I did not get what I have to do. Please could you explain me once again.

**Comment by sairish2001** [comment]
I am saying to made changes into updated files

**Comment by DEEPIKA-SUK** [comment]
Ok sir.

**Comment by DEEPIKA-SUK** [comment]
Sir I have sent you the PR.

**Comment by sairish2001** [comment]
Still not getting required output

![Screenshot_20210309-150751_Chrome.jpg](https://user-images.githubusercontent.com/60106112/110450483-40ede580-80e9-11eb-866e-6786d2ac0c19.jpg)

kindly replace √ with sqrt(

**Comment by DEEPIKA-SUK** [comment]
![Screenshot (117)](https://user-images.githubusercontent.com/64724039/110452345-1ac94500-80eb-11eb-93fd-3d062000a820.png)
Sir I am getting correct output only for the code that I have written.

**Comment by DEEPIKA-SUK** [comment]
Sir I also checked the output for input that you gave after cloning the main repo just now and its showing the correct output that is 12.

**Comment by DEEPIKA-SUK** [comment]
![Screenshot (118)](https://user-images.githubusercontent.com/64724039/110453827-9bd50c00-80ec-11eb-8450-e6c5da7e1f49.png)
Sir its also showing the correct output when I calculated the same on https://sairish2001.github.io/makesmatheasy.github.io/

**Comment by DEEPIKA-SUK** [comment]
Sir the square root function is not working on mobile. I really don't know why its not working. It is working fine on PC but not on mobile. I will try to correct that.

**Comment by DEEPIKA-SUK** [comment]
![Screenshot (121)](https://user-images.githubusercontent.com/64724039/110466516-e01bd880-80fb-11eb-9e82-c0a6725102b6.png)
![WhatsApp Image 2021-03-09 at 17 14 27](https://user-images.githubusercontent.com/64724039/110466526-e14d0580-80fb-11eb-8c25-3b2c602ad42a.jpeg)
![WhatsApp Image 2021-03-09 at 17 14 27 (1)](https://user-images.githubusercontent.com/64724039/110466528-e1e59c00-80fb-11eb-9806-bae62b4cd75a.jpeg)
![Screenshot (122)](https://user-images.githubusercontent.com/64724039/110466529-e27e3280-80fb-11eb-9fd1-d9c1806c318e.png)
Sir kindly review this. I changed '√' with 'sqrt' now.
I have attached screenshots for the both mobile and laptop.
Should I make the PR?
